src/layout/useLayout.ts

```diff
@@ -278,6 +302,7 @@ export const useLayout = ({
     scrollXY,
     positionCanvas,
     fitCanvas,
-    setScrollXY
+    fitToNode,
+    setScrollXY: scrollToXY
```

Comment:
So now this exported `setScrollXY` is now actually calling that function instead of directly setting state, but it's doin the same thing and works the same